### PR TITLE
chore: ini documentation

### PR DESCRIPTION
**Pull request by chrishogg**
----

Notes

----

## PR Checklist
- [ ] Documentation has been checked by at least one team member
- [x] Links work to navigate documentation
- [ ] Images stored in `_images`
- [ ] Spell checked

> If some of the above are not completed why?
> **REASON HERE**

### CODE REVIEW

File: docs/resource-types/dairy/milk-collection.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        - api
+        - data
+    post_tag:
+        - api
+
+---
+
+# Resource Type: Dairy
+
+- [URLs]()
```
Comment (daniel-buchanan) [inline docs/resource-types/dairy/milk-collection.md:18]:
```suggestion
- [URLs](#urls)
```

File: docs/resource-types/dairy/milk-collection.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+- [Response Structure]()
```
Comment (daniel-buchanan) [inline docs/resource-types/dairy/milk-collection.md:19]:
```suggestion
- [Response Structure](#response-structure)
```

File: docs/resource-types/index.md [hunk 3]
```diff
@@ -69,7 +69,9 @@ The Data API is based on the REST standard, where URLs represent collections of
       Provides information about Animal Deaths for individual Animals.
     - [Consignment](/resource-types/livestock/movements/consignment.md)
       Provides information about animal movements that is common accross the other movement resource types
-
+- [Dairy](/resource-types/dairy
```
Comment (daniel-buchanan) [inline docs/resource-types/index.md:72]:
```suggestion
- [Dairy](/resource-types/dairy)
```

**Review by daniel-buchanan — CHANGES_REQUESTED** [review]
Some minor changes.
Question about the analysis @chrishogg these look like they would be common across several resource types? Or is that something we should look at in future?

**Comment by VicXue** [comment]
> Some minor changes. Question about the analysis @chrishogg these look like they would be common across several resource types? Or is that something we should look at in future?

I took this from Chris as he is busy with other changes due to new schema structure. Since we have already decided that milk collection would inherit the load resource json schema, I don't see why we can't re-use some of the existing model definitions.

**Review by daniel-buchanan — APPROVED** [review]

**Pull request merged**